src/utils/emlTemplate.j2

```diff
@@ -39,6 +44,7 @@
             <!-- <role> is used by associatedParty, but its a single item not a list -->
             <!-- <role></role> -->
             <userId directory="http://orcid.org/">{{ contact.orcid }}</userId>
+            <!-- NOTE wondering whether contact.orcid should be contact.indOrcid -->
```

Comment:
Seems similar to contact.lastName, contact.givenName in that I think an ORCID can only apply to an individual so you may not need the 'ind'?

Reply:
Yeah perhaps, I was mainly wondering whether it should be contact.indOrcid because indOrcid is listed [here](https://github.com/cioos-siooc/metadata-entry-form/blob/main/src/utils/blankRecord.js)

Reply:
yes can only use variables names from that link